sway/commands/layout.c

```diff
@@ -162,6 +162,13 @@ struct cmd_results *cmd_layout(int argc, char **argv) {
 			}
 			container->layout = new_layout;
 			container_update_representation(container);
+		} else if (config->handler_context.container) {
+			// i3 avoids changing workspace layouts with a new container
+			// https://github.com/i3/i3/blob/3cd1c45eba6de073bc4300eebb4e1cc1a0c4479a/src/con.c#L1817
+			container = workspace_wrap_children(workspace);
+			container->layout = new_layout;
+			container_update_representation(container);
+			workspace_update_representation(workspace);
```

Comment:
I think this call is implied in `container_update_representation`, is it not? https://github.com/swaywm/sway/blob/39328ca4e4b97bd76153869f6a0a20fef3c17112/sway/tree/container.c#L612-L616